tests/test_target.py

```diff
@@ -14,17 +14,30 @@
 # limitations under the License.
 #
 
+''' Module for testing targets. '''
+
 import os
 import shutil
 import tempfile
 from unittest import TestCase
 
 from devlib import LocalLinuxTarget
+from devlib.utils.misc import load_struct_from_yaml
 
 
 class TestReadTreeValues(TestCase):
+    ''' Class testing Target.read_tree_values_flat() '''
+
+    TARGETS_CONFIG_FILE = 'tests/target_configs.yaml'
```

Comment:
Does pytest guarantees the current folder when running tests ? If not this could be made relative to `__file__`'s folder for example.

Reply:
I run tests under `/devlib` dir. Will use `__file__`'s folder to ensure that.